docs/enhancements/feature-enablement.md

```diff
@@ -0,0 +1,78 @@
+---
+title: feature-enablement
+authors:
+  - "@nmagnezi"
+creation-date: 2022-05-02
+last-updated: 2022-05-02
+---
+
+# Feature Enablement
+
+The current OCM-based authorization scheme used by assisted service as a part of the SaaS operation does not support a rollout of certain features to a select number of users.
+We wish to allow the following:
+* An ability to expose selected group(s) of users to assisted-service functionality (in OCM, the term is `capability`).
+* The user grouping should be based on the organization ID.
+
+
+## Motivation
+
+Some features might be of interest to partners or early adopters.
```

Comment:
Some features also may not be ready for public consumption.

Reply:
done